doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -70,18 +70,17 @@ $ docker-compose up -d
 
 `docker-compose-initializer` will generate a root CA for devices, a key pair for Housekeeping, and a self-signed certificate for the broker (note: this is a *really* bad idea in production). You can tune the compose file further to use legitimate certificates and custom keys, but this is out of the scope of this tutorial.
 
-Compose might take some time to bring everything up, but usually within a minute from the containers creation Astarte will be ready. Compose will forward the following ports to your machine:
+Compose might take some time to bring everything up, but usually within a minute from the containers creation Astarte will be ready.
+It will bring up a Traefik instance to expose Astarte services. You can reach Astarte at the following addresses:
 
-* `4000`: Realm Management API
-* `4001`: Housekeeping API
-* `4002`: AppEngine API
-* `4003`: Pairing API
-* `4040`: Dashboard
-* `8883`: MQTTS
-* `1885`: MQTT with Proxy Protocol for SSL termination (won't be used)
-* `80`: Let's Encrypt verification (won't be used)
+* `api.localhost`: Astarte API
```

Comment:
Maybe it's still worth mentioning the services URLs, expecially pairing, considering it's used by stream-test Qt5